org.mozilla.Firefox.json

```diff
@@ -0,0 +1,136 @@
+{
+  "app-id": "org.mozilla.Firefox",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.22",
+  "sdk": "org.gnome.Sdk",
+  "command": "firefox",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--socket=wayland",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--filesystem=home"
+  ],
+  "modules": [
+    {
+      "name": "autoconf",
+      "config-opts": [
+        "--program-suffix=2.13"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://ftp.gnu.org/gnu/autoconf/autoconf-2.13.tar.gz",
+          "sha256": "f0611136bee505811e9ca11ca7ac188ef5323a8e2ef19cffd3edb3cf08fd791e"
+        }
+      ]
+    },
+    {
+      "name": "gconf",
+      "config-opts": [
+        "--disable-static",
+        "--disable-orbit"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "config-opts": [
+            "--enable-nls",
+            "--with-ssl=openssl"
+          ],
+          "url": "https://download.gnome.org/sources/GConf/3.2/GConf-3.2.6.tar.xz",
+          "sha256": "1912b91803ab09a5eed34d364bf09fe3a2a9c96751fde03a4e0cfa51a04d784c"
+        }
+      ]
+    },
+    {
+      "name": "yasm",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz",
+          "sha256": "3dce6601b495f5b3d45b59f7d2492a340ee7e84b5beca17e48f862502bd5603f"
+        }
+      ]
+    },
+    {
+      "name": "libvpx",
+      "config-opts": [
+        "--enable-runtime-cpu-detect",
+        "--enable-shared",
+        "--enable-pic",
+        "--enable-vp8",
+        "--enable-postproc",
+        "--enable-vp9",
+        "--enable-vp9-highbitdepth",
+        "--enable-experimental",
+        "--enable-spatial-svc"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/webmproject/libvpx/archive/v1.6.0.tar.gz",
+          "sha256": "e2fc00c9f60c76f91a1cde16a2356e33a45b76a5a5a1370df65fd57052a4994a"
+        }
+      ]
+    },
+    {
+      "name": "gstreamer",
+      "config-opts": [
+        "--disable-static"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.10.2.tar.xz",
+          "sha256": "150e8e81febac94c161d8141cde78a38038a8f56e8ec549f353da54994278d65"
+        }
+      ]
+    },
+    {
+      "name": "gst-plugins-base",
+      "config-opts": [
+        "--disable-static"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.10.2.tar.xz",
+          "sha256": "fbc0d40fcb746d2efe2ea47444674029912f66e6107f232766d33b722b97de20"
+        }
+      ]
+    },
```

Comment:
Everything that's not from Sdk should have some `cleanup` directives. We already have some cleaning up to do, we shouldn't increase that debt.